### ISSUE TITLE
Strange blue bloom artifacts in dark scenes 

### ISSUE THREAD

**Issue by ginomessmer** [Unity-Technologies/PostProcessing#93]
Take a look at these screenshots, those artifacts appear on horizon after disabling sun source.

During night:
![Night](https://cloud.githubusercontent.com/assets/8465892/22839752/0e66ee08-efcb-11e6-9930-d0f1111a688a.png)

Daytime:
![temp](https://cloud.githubusercontent.com/assets/8465892/22839867/78ccbf7a-efcb-11e6-8808-152d9f557d08.png)

**Comment by Kink3d** [comment]
Hey,

We believe this to be a conflict with other third party systems and not necessarily PostFX related. Could you upload a reproduction project for me to take a look please.

Thanks.

**Comment by ginomessmer** [comment]
Hey, 

I don't use any other PostFX assets on my main camera. I switched over to linear color space and these blue artifacts don't occur anymore. My best guess is it's probably related to gamma color space. I'm currently setting up a reproduction project with the same settings I used in my main project, but those blue artifacts don't appear yet, which is quite strange.

I disabled all GameObjects in my main project except my main camera with Post Processing Stack, those artifacts are still there although I don't got any light sources enabled. I attached my behavior asset just in case, I'll test whether I can continue to reproduce it.

[Broken.zip](https://github.com/Unity-Technologies/PostProcessing/files/771519/Broken.zip)
